### PR TITLE
CB-3-features

### PR DESCRIPTION
**Pull request by Stacey-Masi**
* **Please check if the PR fulfills these requirements**
- [X ] The pull request title contains the ticket number in the title. For example [CB-1]
Yes, the title is in both the branch name and the pull request name.

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
The pull request is the features component for the project. 
It contains free, open-sourced pictures for each feature as well as a header, subtitle, and description for each feature. 
Jumbotron was added to match the viewing of other components in the project.

* **What is the current behavior?What does the code do before you made the new changes?** 
It is very basic and not as detailed. There are no images.

* **What is the new behavior?**
A constant card was implemented to reduce redundancy and increase readability. 
Each feature aspect can be altered later on in development. 
There is an optional property currently included which slightly affects splitscreen viewing but makes the images even when on fullscreen viewing. When this property is excluded, one feature card is slightly longer on both fullscreen and splitscreen.
It is more visually pleasing.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No breaking change. It's just prettier :)

* **Other information**:
Image of features component
featurescomponent_with_theme

### CODE REVIEW

File: frontend/src/components/LandingPage/Features/Features.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        - and getting videos to go along with your recipe
+*/
+
+import React from 'react';
+import { Card, CardImg, CardTitle, CardText, CardDeck, CardSubtitle, CardBody, Jumbotron } from 'reactstrap';
+import './Features.css'
+import camera from './assets/coolcamera.jpeg';
+import nutritional from './assets/nutritional.jpg';
+import recipes from './assets/goodrecipe.jpeg';
+import videos from './assets/videos.jpg';
```
Comment (sunghyun96) [inline frontend/src/components/LandingPage/Features/Features.js]: Passing on some feedback I got from Miguel earlier on my tools component images!

Images could be moved to the public folder and then you can just have the filepath like `src='/coolcamera.jpeg'` or maybe you could have something like `const camera = '/coolcamera.jpeg'` if you wanted to keep the `img = {camera}` down below as is.

**Review by sunghyun96 — CHANGES_REQUESTED** [review]
Everything looks great Stacey! I did leave a comment for a part of the Features.js file regarding some feedback I got myself from Miguel earlier, but great work overall!

File: frontend/src/components/LandingPage/Features/Features.js (hunk 1 above)
Comment (Stacey-Masi) [reply frontend/src/components/LandingPage/Features/Features.js]: That sounds great! I'll implement those changes

**Review by Stacey-Masi — COMMENTED** [review]
fixed photo location

**Review by sunghyun96 — APPROVED** [review]

File: frontend/src/components/LandingPage/Features/Features.js (hunk 1 above)
Comment (sunghyun96) [reply frontend/src/components/LandingPage/Features/Features.js]: Awesome! Just approved your changes!

File: frontend/src/components/LandingPage/Features/Features.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+import './Features.css'
+const camera = '/coolcamera.jpeg';
+const nutritional = '/nutritional.jpg';
+const recipes = '/goodrecipe.jpeg';
+const videos = '/videos.jpg';
+
+// including id="pictures" makes the picture format stretch when on split screen but even when on full screen
+// removing id="pictures" will make nutrition card slightly longer
+// constant for the card used throughout this file
+const FeaturesCard = (props) => {
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Features/Features.js]: Can we make this a functional component instead of a const.

Comment (Codubee) [reply frontend/src/components/LandingPage/Features/Features.js]: What do you think about putting the FeaturesCard in a separate file and importing it here instead. That way we can have one component per file to minimize confusion.

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Stacey! Love the photos you chose. Left some comments that we can talk about.

File: frontend/src/components/LandingPage/Features/Features.js (hunk 2 above)
Comment (Stacey-Masi) [reply frontend/src/components/LandingPage/Features/Features.js]: Absolutely! I created the file for the FeaturesCard function and exported it to the Features function just now

**Review by AdemOdza — APPROVED** [review]
Looks great! You have already implemented the suggestions from Miguel and Sung and I don't see anything that I would change.

**Review by saitanjavuru — APPROVED** [review]
Everything looks good!

**Review by alekhyapinnamaneni — APPROVED** [review]
This looks great! The only thing I might suggest is to change background/text color to go with the dark green and cream/white theme we selected.

**Review by Codubee — APPROVED** [review]
LGTM

**Pull request merged**